components/transactions/TransactionItem.js

```diff
@@ -154,6 +156,36 @@ const TransactionItem = ({ displayActions, collective, transaction, onMutationSu
 
   const displayedAmount = getDisplayedAmount(transaction, collective);
 
+  const transactionDetailLink = transactionKind => {
```

Comment:
I think you don't need a parameter here, you already have `kind` available in the closure.

Reply:
The parameter is removed. 👍🏽